provisioning/Prospero_update_to_version.adoc

```diff
@@ -0,0 +1,178 @@
+---
+categories:
+ - provisioning
+stability-level: default
+# Specify the Feature Development tracker issue for the feature.
+# This must be an issue tracked in https://github.com/orgs/wildfly/projects/7/views/1.
+# To create a Feature Development tracker issue, go to  https://github.com/wildfly/wildfly-proposals/issues/new/choose
+# and select 'Feature Development'
+issue: https://github.com/wildfly/wildfly-proposals/issues/705
+# Provide the github ids of the members of the feature team, organized by role.
+# Provide a single id for the 'assignee' role. Use a yaml list for the 'sme' and
+# 'outside-perspective' roles, even if there is only one person in a role.
+feature-team:
+ developer: Pedro Hos
+ sme: 
+  - Jeff Mesnil
+ outside-perspective:
+  - Vojtech Salbaba
+  - Symone Simpkins
+# This should be blank during initial development of a feature. It may be used
+# after the feature is completed if a subsequent issue is field to track promotion
+# of this feature to a higher stability level
+promoted-by:
+---
+= Add ability to update WildFly to a specific version of a manifest using wildfly-channels.
+:author:            Bartosz Spyrko-Smietanko
+:email:             bspyrkos@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+__<The entire document should be one to two pages long. We will write each analysis document as if it is a conversation with a future developer. This requires a good writing style, with full sentences organized into paragraphs. Bullets are acceptable only for visual style, not as an excuse for writing sentence fragments.>__
+
+== Overview
+
+Currently, Prospero update operation always picks the latest available manifest(s) version(s) to update to. Some users might prefer to perform an update to a concrete version that they can pre-test.
+
+This proposal adds a new parameter to the `update perform` and `update prepare` operations that will accept a version of a manifest version.
+
+=== User Stories
+
+ - As an admin I want to use a logical or physical manifest version to update the server to a specific version.
+ - As an admin I want to be able to perform a downgrade if a specified version is lower than a current version. Such operation should result in a warning that the user has to accept.
+ - As an admin I want to be able to perform a downgrade in a non-interactive way by specifying a command-line flag.
+ - As an admin I want to be prevented from downgrading a manifest version unless the new version is specified explicitly.
+
+
+==== Nice-to-have:
+ - As an admin customer I want to be able to see a list of potential manifest versions I can update (or downgrade to).
+
+== Issue Metadata
+
+=== Related Issues
+
+ - https://github.com/wildfly-extras/prospero/issues/754
```

Comment:
Could you also add a link to https://github.com/wildfly/wildfly-proposals/pull/577 please?